crates/bevy_render/src/primitives/mod.rs

```diff
@@ -95,6 +95,35 @@ impl Aabb {
     pub fn max(&self) -> Vec3A {
         self.center + self.half_extents
     }
+
+    /// Check if the point is at the front side of the plane.
+    /// Referenced from: [AABB Plane intersection](https://gdbooks.gitbooks.io/3dcollisions/content/Chapter2/static_aabb_plane.html)
+    #[inline]
+    pub fn is_forward_plane(
+        &self,
+        p_normal: &Vec3A,
+        p_distance: f32,
+        world_from_local: &Affine3A,
+    ) -> bool {
+        let aabb_center_world = world_from_local.transform_point3a(self.center);
+        let bb_axis_x_world = world_from_local.x_axis * self.half_extents.x;
+        let bb_axis_y_world = world_from_local.y_axis * self.half_extents.y;
+        let bb_axis_z_world = world_from_local.z_axis * self.half_extents.z;
+        let half_extents_x_world = bb_axis_x_world.dot(Vec3A::X).abs()
+            + bb_axis_y_world.dot(Vec3A::X).abs()
+            + bb_axis_z_world.dot(Vec3A::X).abs();
+        let half_extents_y_world = bb_axis_x_world.dot(Vec3A::Y).abs()
+            + bb_axis_y_world.dot(Vec3A::Y).abs()
+            + bb_axis_z_world.dot(Vec3A::Y).abs();
+        let half_extents_z_world = bb_axis_x_world.dot(Vec3A::Z).abs()
+            + bb_axis_y_world.dot(Vec3A::Z).abs()
+            + bb_axis_z_world.dot(Vec3A::Z).abs();
+        let r = half_extents_x_world * p_normal.x.abs()
+            + half_extents_y_world * p_normal.y.abs()
+            + half_extents_z_world * p_normal.z.abs();
```

Comment:
By similar operations here, you can simplify this down to:
```suggestion
        let r = (world_from_local.matrix3 * self.half_extents).dot(p_normal.abs())
```
Although I'm not sure if the absolute value is in a correct spot here, I'll think about it more, I've been just doing algebra so far.